manifest.json

```diff
@@ -10,6 +10,7 @@
   },
   "permissions": [
     "history",
```

Comment:
Do we still need the `history` permission? Maybe we do, if we want to mass-import browser history later?